corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -106,7 +107,13 @@ def run(self):
         pillow_logging.info("Starting pillow %s" % self.__class__)
         with configure_scope() as scope:
             scope.set_tag("pillow_name", self.get_name())
-        self.process_changes(since=self.get_last_checkpoint_sequence(), forever=True)
+        if self.process_num==0  and self.is_dedicated_migration_process:
```

Comment:
nit: stickler is not going to like this spacing

Reply:
Noted Danny. thanks